requirements/extra-pdf-image.txt

```diff
@@ -219,7 +217,7 @@ typing-extensions==4.8.0
     #   torch
 tzdata==2023.3
     # via pandas
-unstructured-inference==0.5.31
+unstructured-inference==0.5.28
```

Comment:
hmm, this downgrades unstructured-inference? feel like I remember someone mentioning changes to constraints recently, guessing this isn't what we want here..

Reply:
Thanks for catching, I have changed it back to latest version